src/Avalonia.Base/Visual.cs

```diff
@@ -36,12 +36,7 @@ public class Visual : StyledElement
         /// </summary>
         public static readonly DirectProperty<Visual, Rect> BoundsProperty =
             AvaloniaProperty.RegisterDirect<Visual, Rect>(nameof(Bounds), o => o.Bounds);
-
-        public static readonly DirectProperty<Visual, TransformedBounds?> TransformedBoundsProperty =
-            AvaloniaProperty.RegisterDirect<Visual, TransformedBounds?>(
-                nameof(TransformedBounds),
-                o => o.TransformedBounds);
-
```

Comment:
Wonder if instead of removing this it should be implemented in the composition renderer: https://github.com/AvaloniaUI/Avalonia/issues/9569?

Reply:
See this diff https://github.com/AvaloniaUI/Avalonia/pull/10078/files#diff-75a2cfdea0f7dde6a572ef293c1b72b466cb27380dd4b2c5d1b71f579d08b4e8

It seems there is an alternative now with compositor https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Rendering/Composition/Visual.cs#L36-L52
Keeping in mind that this property can't be observable anymore.

But it's internal now, which I am not sure why. @kekekeks